--- a/ui-text/src/commonMain/kotlin/androidx/compose/ui/text/platform/Synchronization.kt
+++ b/ui-text/src/commonMain/kotlin/androidx/compose/ui/text/platform/Synchronization.kt
@@ -16,15 +16,13 @@
 
 package androidx.compose.ui.text.platform
 
-internal expect class SynchronizedObject
+expect class SynchronizedObject
 
 /**
  * Returns [ref] as a [SynchronizedObject] on platforms where [Any] is a valid [SynchronizedObject],
  * or a new [SynchronizedObject] instance if [ref] is null or this is not supported on the current
  * platform.
  */
-internal expect inline fun makeSynchronizedObject(ref: Any? = null): SynchronizedObject
+expect inline fun makeSynchronizedObject(ref: Any? = null): SynchronizedObject
 
-@PublishedApi
-@Suppress("LESS_VISIBLE_TYPE_ACCESS_IN_INLINE_WARNING") // b/446705238
-internal expect inline fun <R> synchronized(lock: SynchronizedObject, block: () -> R): R
+expect inline fun <R> synchronized(lock: SynchronizedObject, block: () -> R): R
